TASVideos/Pages/Forum/Topics/Index.cshtml

```diff
@@ -137,5 +144,14 @@
 	   class="btn btn-primary btn-sm float-end">
 		<span class="fa fa-plus"></span> Post
 	</a>
-	<partial name="_TopicCrumbs" model="Model.Topic" />
+	<nav aria-label="breadcrumb" class="card card-header d-block mb-2">
+		<ol class="breadcrumb m-0">
+			<li class="breadcrumb-item"><a asp-page="/Forum/Index">Forum</a></li>
+			<li class="breadcrumb-item"><a asp-page="/Forum/Subforum/Index" asp-route-id="@Model.Topic.ForumId">@Model.Topic.ForumName</a></li>
+			<li class="breadcrumb-item active" aria-current="page">
+				<p>@(Model.Topic.Title)</p>
```

Comment:
So, doing a \<p\> here gives this a 1rem bottom margin, and that looks off.

![image](https://user-images.githubusercontent.com/22375320/152538844-db67f79d-78d4-487b-b5e1-3e22532a736d.png)
Example on the top of the page:
![image](https://user-images.githubusercontent.com/22375320/152539121-d41b4ddc-e918-4ba1-8bc3-ced2e315779c.png)